src/core/packages/chrome/browser-internal/src/chrome_service.tsx

```diff
@@ -212,6 +213,7 @@ export class ChromeService {
     const badge$ = new BehaviorSubject<ChromeBadge | undefined>(undefined);
     const customNavLink$ = new BehaviorSubject<ChromeNavLink | undefined>(undefined);
     const helpSupportUrl$ = new BehaviorSubject<string>(docLinks.links.kibana.askElastic);
+    const appMenu$ = new BehaviorSubject<AppMenuConfig | undefined>(undefined);
```

Comment:
afaik the currentActionMenu$ is reset every time the app changes. don't we need to do the same for appMenu?